composer.json

```diff
@@ -17,7 +17,7 @@
         "php": "^5.3.6 || >=7.0 <7.2",
         "ext-tokenizer": "*",
         "sebastian/diff": "^1.1",
-        "symfony/console": "^2.3 || ^3.0",
+        "symfony/console": "^2.4 || ^3.0",
```

Comment:
2.3 is supported to May this year (http://symfony.com/roadmap?version=2.3#checker), would be nice if we could keep it (especially if the up is only for test). However if this is a lot of work, than up'ping is :+1:  with me